admin/tabs/cluster/ClusterTabModel.ts

```diff
@@ -57,20 +60,28 @@ export class ClusterTabModel extends HoistModel {
             let data = await XH.fetchJson({url: 'clusterAdmin/allInstances', loadSpec});
             data = data.map(row => ({
                 ...row,
+                isLocal: row.name == XH.environmentService.serverInstance,
                 usedHeapMb: row.memory?.usedHeapMb,
                 usedPctMax: row.memory?.usedPctMax
             }));
 
             gridModel.loadData(data);
             await gridModel.preSelectFirstAsync();
         } catch (e) {
+            gridModel.clear();
             XH.handleException(e);
         }
     }
 
     constructor() {
         super();
 
+        this.timer = Timer.create({
+            runFn: this.autoRefreshAsync,
```

Comment:
Not sure about continuing to fire this when the tab is not visible, especially since it has a modal exception handler.

Should we make this conditional upon tab visibility?

Reply:
Makes sense - would probably want to do the same for `MonitorTabModel` then?

Reply:
Monitor has special toast behavior I believe - deliberately refreshing and alerting admin who is looking at app that monitor status has changed

Reply:
although arguably that should not be dependent on have you loaded tab